Add files via upload

KeatonSushi opened the pull request:
added temporary images to the about us page and added content from old site.

LitterPic commented on aboutUsNew.html:
```diff
@@ -1,86 +1,176 @@
-              </li>
-            </ul>
-          </div>
+      <div class="container-fluid">
```
I have a navbar.html and an include-navbar.js defined.  Are you defining a new navbar?

KeatonSushi commented:
I did a copy and paste from the index.html file you had.

On Wed, Apr 19, 2023, 4:32 PM Alek ***@***.***> wrote:

> ***@***.**** commented on this pull request.
> ------------------------------
>
> In aboutUsNew.html
> <https://github.com/LitterPic/litterpic_web/pull/7#discussion_r1171875248>
> :
>
> > -            <ul class="navbar-nav ms-auto">
> -              <li class="nav-item">
> -                <a class="nav-link" aria-current="page" href="#">Home</a>
> -              </li>
> -              <li class="nav-item">
> -                <a class="nav-link" href="#">Events</a>
> -              </li>
> -              <li class="nav-item">
> -                <a class="nav-link" href="#">Donate</a>
> -              </li>
> -              <li class="nav-item">
> -                <a class="nav-link" href="#">About Us</a>
> -              </li>
> -            </ul>
> -          </div>
> +      <div class="container-fluid">
>
> I have a navbar.html and an include-navbar.js defined. Are you defining a
> new navbar?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/LitterPic/litterpic_web/pull/7#pullrequestreview-1392914761>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A4ABJWWLZKPWFNIOZWE63UDXCBKWHANCNFSM6AAAAAAXD5ANNY>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

LitterPic commented:
Sorry I’m not understanding.  I have a navbar set up that is shared across all screens, it looks like you’re implementing a new nav bar in a page called AboutUsNew.html

> On Apr 19, 2023, at 5:43 PM, KeatonSushi ***@***.***> wrote:
> 
> 
> I did a copy and paste from the index.html file you had.
> 
> On Wed, Apr 19, 2023, 4:32 PM Alek ***@***.***> wrote:
> 
> > ***@***.**** commented on this pull request.
> > ------------------------------
> >
> > In aboutUsNew.html
> > <https://github.com/LitterPic/litterpic_web/pull/7#discussion_r1171875248>
> > :
> >
> > > - <ul class="navbar-nav ms-auto">
> > - <li class="nav-item">
> > - <a class="nav-link" aria-current="page" href="#">Home</a>
> > - </li>
> > - <li class="nav-item">
> > - <a class="nav-link" href="#">Events</a>
> > - </li>
> > - <li class="nav-item">
> > - <a class="nav-link" href="#">Donate</a>
> > - </li>
> > - <li class="nav-item">
> > - <a class="nav-link" href="#">About Us</a>
> > - </li>
> > - </ul>
> > - </div>
> > + <div class="container-fluid">
> >
> > I have a navbar.html and an include-navbar.js defined. Are you defining a
> > new navbar?
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <https://github.com/LitterPic/litterpic_web/pull/7#pullrequestreview-1392914761>,
> > or unsubscribe
> > <https://github.com/notifications/unsubscribe-auth/A4ABJWWLZKPWFNIOZWE63UDXCBKWHANCNFSM6AAAAAAXD5ANNY>
> > .
> > You are receiving this because you authored the thread.Message ID:
> > ***@***.***>
> >
> —
> Reply to this email directly, view it on GitHub <https://github.com/LitterPic/litterpic_web/pull/7#issuecomment-1515423456>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AZYPMLYKJSQJWIMAO27FQN3XCBL7NANCNFSM6AAAAAAXD5ANNY>.
> You are receiving this because you commented.
>

KeatonSushi commented:
Oh okay. How do I copy and paste your original navbar?

On Wed, Apr 19, 2023, 4:49 PM Alek ***@***.***> wrote:

> Sorry I’m not understanding. I have a navbar set up that is shared across
> all screens, it looks like you’re implementing a new nav bar in a page
> called AboutUsNew.html
>
> > On Apr 19, 2023, at 5:43 PM, KeatonSushi ***@***.***> wrote:
> >
> >
> > I did a copy and paste from the index.html file you had.
> >
> > On Wed, Apr 19, 2023, 4:32 PM Alek ***@***.***> wrote:
> >
> > > ***@***.**** commented on this pull request.
> > > ------------------------------
> > >
> > > In aboutUsNew.html
> > > <
> https://github.com/LitterPic/litterpic_web/pull/7#discussion_r1171875248>
> > > :
> > >
> > > > - <ul class="navbar-nav ms-auto">
> > > - <li class="nav-item">
> > > - <a class="nav-link" aria-current="page" href="#">Home</a>
> > > - </li>
> > > - <li class="nav-item">
> > > - <a class="nav-link" href="#">Events</a>
> > > - </li>
> > > - <li class="nav-item">
> > > - <a class="nav-link" href="#">Donate</a>
> > > - </li>
> > > - <li class="nav-item">
> > > - <a class="nav-link" href="#">About Us</a>
> > > - </li>
> > > - </ul>
> > > - </div>
> > > + <div class="container-fluid">
> > >
> > > I have a navbar.html and an include-navbar.js defined. Are you
> defining a
> > > new navbar?
> > >
> > > —
> > > Reply to this email directly, view it on GitHub
> > > <
> https://github.com/LitterPic/litterpic_web/pull/7#pullrequestreview-1392914761
> >,
> > > or unsubscribe
> > > <
> https://github.com/notifications/unsubscribe-auth/A4ABJWWLZKPWFNIOZWE63UDXCBKWHANCNFSM6AAAAAAXD5ANNY
> >
> > > .
> > > You are receiving this because you authored the thread.Message ID:
> > > ***@***.***>
> > >
> > —
> > Reply to this email directly, view it on GitHub <
> https://github.com/LitterPic/litterpic_web/pull/7#issuecomment-1515423456>,
> or unsubscribe <
> https://github.com/notifications/unsubscribe-auth/AZYPMLYKJSQJWIMAO27FQN3XCBL7NANCNFSM6AAAAAAXD5ANNY
> >.
> > You are receiving this because you commented.
> >
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/LitterPic/litterpic_web/pull/7#issuecomment-1515429448>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A4ABJWWSRXP2UTD7FCGSTDDXCBMXBANCNFSM6AAAAAAXD5ANNY>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

KeatonSushi commented:
Or where should I copy it from rather.

On Wed, Apr 19, 2023, 4:51 PM Keaton Milliner ***@***.***> wrote:

> Oh okay. How do I copy and paste your original navbar?
>
> On Wed, Apr 19, 2023, 4:49 PM Alek ***@***.***> wrote:
>
>> Sorry I’m not understanding. I have a navbar set up that is shared across
>> all screens, it looks like you’re implementing a new nav bar in a page
>> called AboutUsNew.html
>>
>> > On Apr 19, 2023, at 5:43 PM, KeatonSushi ***@***.***> wrote:
>> >
>> >
>> > I did a copy and paste from the index.html file you had.
>> >
>> > On Wed, Apr 19, 2023, 4:32 PM Alek ***@***.***> wrote:
>> >
>> > > ***@***.**** commented on this pull request.
>> > > ------------------------------
>> > >
>> > > In aboutUsNew.html
>> > > <
>> https://github.com/LitterPic/litterpic_web/pull/7#discussion_r1171875248>
>> > > :
>> > >
>> > > > - <ul class="navbar-nav ms-auto">
>> > > - <li class="nav-item">
>> > > - <a class="nav-link" aria-current="page" href="#">Home</a>
>> > > - </li>
>> > > - <li class="nav-item">
>> > > - <a class="nav-link" href="#">Events</a>
>> > > - </li>
>> > > - <li class="nav-item">
>> > > - <a class="nav-link" href="#">Donate</a>
>> > > - </li>
>> > > - <li class="nav-item">
>> > > - <a class="nav-link" href="#">About Us</a>
>> > > - </li>
>> > > - </ul>
>> > > - </div>
>> > > + <div class="container-fluid">
>> > >
>> > > I have a navbar.html and an include-navbar.js defined. Are you
>> defining a
>> > > new navbar?
>> > >
>> > > —
>> > > Reply to this email directly, view it on GitHub
>> > > <
>> https://github.com/LitterPic/litterpic_web/pull/7#pullrequestreview-1392914761
>> >,
>> > > or unsubscribe
>> > > <
>> https://github.com/notifications/unsubscribe-auth/A4ABJWWLZKPWFNIOZWE63UDXCBKWHANCNFSM6AAAAAAXD5ANNY
>> >
>> > > .
>> > > You are receiving this because you authored the thread.Message ID:
>> > > ***@***.***>
>> > >
>> > —
>> > Reply to this email directly, view it on GitHub <
>> https://github.com/LitterPic/litterpic_web/pull/7#issuecomment-1515423456>,
>> or unsubscribe <
>> https://github.com/notifications/unsubscribe-auth/AZYPMLYKJSQJWIMAO27FQN3XCBL7NANCNFSM6AAAAAAXD5ANNY
>> >.
>> > You are receiving this because you commented.
>> >
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/LitterPic/litterpic_web/pull/7#issuecomment-1515429448>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/A4ABJWWSRXP2UTD7FCGSTDDXCBMXBANCNFSM6AAAAAAXD5ANNY>
>> .
>> You are receiving this because you authored the thread.Message ID:
>> ***@***.***>
>>
>

KeatonSushi commented:
I fixed the navbar.

On Wed, Apr 19, 2023, 4:52 PM Keaton Milliner ***@***.***> wrote:

> Or where should I copy it from rather.
>
> On Wed, Apr 19, 2023, 4:51 PM Keaton Milliner ***@***.***>
> wrote:
>
>> Oh okay. How do I copy and paste your original navbar?
>>
>> On Wed, Apr 19, 2023, 4:49 PM Alek ***@***.***> wrote:
>>
>>> Sorry I’m not understanding. I have a navbar set up that is shared
>>> across all screens, it looks like you’re implementing a new nav bar in a
>>> page called AboutUsNew.html
>>>
>>> > On Apr 19, 2023, at 5:43 PM, KeatonSushi ***@***.***> wrote:
>>> >
>>> >
>>> > I did a copy and paste from the index.html file you had.
>>> >
>>> > On Wed, Apr 19, 2023, 4:32 PM Alek ***@***.***> wrote:
>>> >
>>> > > ***@***.**** commented on this pull request.
>>> > > ------------------------------
>>> > >
>>> > > In aboutUsNew.html
>>> > > <
>>> https://github.com/LitterPic/litterpic_web/pull/7#discussion_r1171875248
>>> >
>>> > > :
>>> > >
>>> > > > - <ul class="navbar-nav ms-auto">
>>> > > - <li class="nav-item">
>>> > > - <a class="nav-link" aria-current="page" href="#">Home</a>
>>> > > - </li>
>>> > > - <li class="nav-item">
>>> > > - <a class="nav-link" href="#">Events</a>
>>> > > - </li>
>>> > > - <li class="nav-item">
>>> > > - <a class="nav-link" href="#">Donate</a>
>>> > > - </li>
>>> > > - <li class="nav-item">
>>> > > - <a class="nav-link" href="#">About Us</a>
>>> > > - </li>
>>> > > - </ul>
>>> > > - </div>
>>> > > + <div class="container-fluid">
>>> > >
>>> > > I have a navbar.html and an include-navbar.js defined. Are you
>>> defining a
>>> > > new navbar?
>>> > >
>>> > > —
>>> > > Reply to this email directly, view it on GitHub
>>> > > <
>>> https://github.com/LitterPic/litterpic_web/pull/7#pullrequestreview-1392914761
>>> >,
>>> > > or unsubscribe
>>> > > <
>>> https://github.com/notifications/unsubscribe-auth/A4ABJWWLZKPWFNIOZWE63UDXCBKWHANCNFSM6AAAAAAXD5ANNY
>>> >
>>> > > .
>>> > > You are receiving this because you authored the thread.Message ID:
>>> > > ***@***.***>
>>> > >
>>> > —
>>> > Reply to this email directly, view it on GitHub <
>>> https://github.com/LitterPic/litterpic_web/pull/7#issuecomment-1515423456>,
>>> or unsubscribe <
>>> https://github.com/notifications/unsubscribe-auth/AZYPMLYKJSQJWIMAO27FQN3XCBL7NANCNFSM6AAAAAAXD5ANNY
>>> >.
>>> > You are receiving this because you commented.
>>> >
>>>
>>> —
>>> Reply to this email directly, view it on GitHub
>>> <https://github.com/LitterPic/litterpic_web/pull/7#issuecomment-1515429448>,
>>> or unsubscribe
>>> <https://github.com/notifications/unsubscribe-auth/A4ABJWWSRXP2UTD7FCGSTDDXCBMXBANCNFSM6AAAAAAXD5ANNY>
>>> .
>>> You are receiving this because you authored the thread.Message ID:
>>> ***@***.***>
>>>
>>

LitterPic commented:
I dont see it fixed yet.  We can connect tomorrow if you like.

> On Apr 19, 2023, at 6:35 PM, KeatonSushi ***@***.***> wrote:
> 
> 
> I fixed the navbar.
> 
> On Wed, Apr 19, 2023, 4:52 PM Keaton Milliner ***@***.***> wrote:
> 
> > Or where should I copy it from rather.
> >
> > On Wed, Apr 19, 2023, 4:51 PM Keaton Milliner ***@***.***>
> > wrote:
> >
> >> Oh okay. How do I copy and paste your original navbar?
> >>
> >> On Wed, Apr 19, 2023, 4:49 PM Alek ***@***.***> wrote:
> >>
> >>> Sorry I’m not understanding. I have a navbar set up that is shared
> >>> across all screens, it looks like you’re implementing a new nav bar in a
> >>> page called AboutUsNew.html
> >>>
> >>> > On Apr 19, 2023, at 5:43 PM, KeatonSushi ***@***.***> wrote:
> >>> >
> >>> >
> >>> > I did a copy and paste from the index.html file you had.
> >>> >
> >>> > On Wed, Apr 19, 2023, 4:32 PM Alek ***@***.***> wrote:
> >>> >
> >>> > > ***@***.**** commented on this pull request.
> >>> > > ------------------------------
> >>> > >
> >>> > > In aboutUsNew.html
> >>> > > <
> >>> https://github.com/LitterPic/litterpic_web/pull/7#discussion_r1171875248
> >>> >
> >>> > > :
> >>> > >
> >>> > > > - <ul class="navbar-nav ms-auto">
> >>> > > - <li class="nav-item">
> >>> > > - <a class="nav-link" aria-current="page" href="#">Home</a>
> >>> > > - </li>
> >>> > > - <li class="nav-item">
> >>> > > - <a class="nav-link" href="#">Events</a>
> >>> > > - </li>
> >>> > > - <li class="nav-item">
> >>> > > - <a class="nav-link" href="#">Donate</a>
> >>> > > - </li>
> >>> > > - <li class="nav-item">
> >>> > > - <a class="nav-link" href="#">About Us</a>
> >>> > > - </li>
> >>> > > - </ul>
> >>> > > - </div>
> >>> > > + <div class="container-fluid">
> >>> > >
> >>> > > I have a navbar.html and an include-navbar.js defined. Are you
> >>> defining a
> >>> > > new navbar?
> >>> > >
> >>> > > —
> >>> > > Reply to this email directly, view it on GitHub
> >>> > > <
> >>> https://github.com/LitterPic/litterpic_web/pull/7#pullrequestreview-1392914761
> >>> >,
> >>> > > or unsubscribe
> >>> > > <
> >>> https://github.com/notifications/unsubscribe-auth/A4ABJWWLZKPWFNIOZWE63UDXCBKWHANCNFSM6AAAAAAXD5ANNY
> >>> >
> >>> > > .
> >>> > > You are receiving this because you authored the thread.Message ID:
> >>> > > ***@***.***>
> >>> > >
> >>> > —
> >>> > Reply to this email directly, view it on GitHub <
> >>> https://github.com/LitterPic/litterpic_web/pull/7#issuecomment-1515423456>,
> >>> or unsubscribe <
> >>> https://github.com/notifications/unsubscribe-auth/AZYPMLYKJSQJWIMAO27FQN3XCBL7NANCNFSM6AAAAAAXD5ANNY
> >>> >.
> >>> > You are receiving this because you commented.
> >>> >
> >>>
> >>> —
> >>> Reply to this email directly, view it on GitHub
> >>> <https://github.com/LitterPic/litterpic_web/pull/7#issuecomment-1515429448>,
> >>> or unsubscribe
> >>> <https://github.com/notifications/unsubscribe-auth/A4ABJWWSRXP2UTD7FCGSTDDXCBMXBANCNFSM6AAAAAAXD5ANNY>
> >>> .
> >>> You are receiving this because you authored the thread.Message ID:
> >>> ***@***.***>
> >>>
> >>
> —
> Reply to this email directly, view it on GitHub <https://github.com/LitterPic/litterpic_web/pull/7#issuecomment-1515468072>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AZYPMLYCO3SYW4THTOJ3CM3XCBSE5ANCNFSM6AAAAAAXD5ANNY>.
> You are receiving this because you commented.
>

KeatonSushi commented:
Okay & I updated them in the aboutusNew.html & donateNew.html so I wouldn't
interfere with the regular files you already had done.

On Wed, Apr 19, 2023, 5:45 PM Alek ***@***.***> wrote:

> I dont see it fixed yet. We can connect tomorrow if you like.
>
> > On Apr 19, 2023, at 6:35 PM, KeatonSushi ***@***.***> wrote:
> >
> >
> > I fixed the navbar.
> >
> > On Wed, Apr 19, 2023, 4:52 PM Keaton Milliner ***@***.***> wrote:
> >
> > > Or where should I copy it from rather.
> > >
> > > On Wed, Apr 19, 2023, 4:51 PM Keaton Milliner ***@***.***>
> > > wrote:
> > >
> > >> Oh okay. How do I copy and paste your original navbar?
> > >>
> > >> On Wed, Apr 19, 2023, 4:49 PM Alek ***@***.***> wrote:
> > >>
> > >>> Sorry I’m not understanding. I have a navbar set up that is shared
> > >>> across all screens, it looks like you’re implementing a new nav bar
> in a
> > >>> page called AboutUsNew.html
> > >>>
> > >>> > On Apr 19, 2023, at 5:43 PM, KeatonSushi ***@***.***> wrote:
> > >>> >
> > >>> >
> > >>> > I did a copy and paste from the index.html file you had.
> > >>> >
> > >>> > On Wed, Apr 19, 2023, 4:32 PM Alek ***@***.***> wrote:
> > >>> >
> > >>> > > ***@***.**** commented on this pull request.
> > >>> > > ------------------------------
> > >>> > >
> > >>> > > In aboutUsNew.html
> > >>> > > <
> > >>>
> https://github.com/LitterPic/litterpic_web/pull/7#discussion_r1171875248
> > >>> >
> > >>> > > :
> > >>> > >
> > >>> > > > - <ul class="navbar-nav ms-auto">
> > >>> > > - <li class="nav-item">
> > >>> > > - <a class="nav-link" aria-current="page" href="#">Home</a>
> > >>> > > - </li>
> > >>> > > - <li class="nav-item">
> > >>> > > - <a class="nav-link" href="#">Events</a>
> > >>> > > - </li>
> > >>> > > - <li class="nav-item">
> > >>> > > - <a class="nav-link" href="#">Donate</a>
> > >>> > > - </li>
> > >>> > > - <li class="nav-item">
> > >>> > > - <a class="nav-link" href="#">About Us</a>
> > >>> > > - </li>
> > >>> > > - </ul>
> > >>> > > - </div>
> > >>> > > + <div class="container-fluid">
> > >>> > >
> > >>> > > I have a navbar.html and an include-navbar.js defined. Are you
> > >>> defining a
> > >>> > > new navbar?
> > >>> > >
> > >>> > > —
> > >>> > > Reply to this email directly, view it on GitHub
> > >>> > > <
> > >>>
> https://github.com/LitterPic/litterpic_web/pull/7#pullrequestreview-1392914761
> > >>> >,
> > >>> > > or unsubscribe
> > >>> > > <
> > >>>
> https://github.com/notifications/unsubscribe-auth/A4ABJWWLZKPWFNIOZWE63UDXCBKWHANCNFSM6AAAAAAXD5ANNY
> > >>> >
> > >>> > > .
> > >>> > > You are receiving this because you authored the thread.Message
> ID:
> > >>> > > ***@***.***>
> > >>> > >
> > >>> > —
> > >>> > Reply to this email directly, view it on GitHub <
> > >>>
> https://github.com/LitterPic/litterpic_web/pull/7#issuecomment-1515423456
> >,
> > >>> or unsubscribe <
> > >>>
> https://github.com/notifications/unsubscribe-auth/AZYPMLYKJSQJWIMAO27FQN3XCBL7NANCNFSM6AAAAAAXD5ANNY
> > >>> >.
> > >>> > You are receiving this because you commented.
> > >>> >
> > >>>
> > >>> —
> > >>> Reply to this email directly, view it on GitHub
> > >>> <
> https://github.com/LitterPic/litterpic_web/pull/7#issuecomment-1515429448
> >,
> > >>> or unsubscribe
> > >>> <
> https://github.com/notifications/unsubscribe-auth/A4ABJWWSRXP2UTD7FCGSTDDXCBMXBANCNFSM6AAAAAAXD5ANNY
> >
> > >>> .
> > >>> You are receiving this because you authored the thread.Message ID:
> > >>> ***@***.***>
> > >>>
> > >>
> > —
> > Reply to this email directly, view it on GitHub <
> https://github.com/LitterPic/litterpic_web/pull/7#issuecomment-1515468072>,
> or unsubscribe <
> https://github.com/notifications/unsubscribe-auth/AZYPMLYCO3SYW4THTOJ3CM3XCBSE5ANCNFSM6AAAAAAXD5ANNY
> >.
> > You are receiving this because you commented.
> >
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/LitterPic/litterpic_web/pull/7#issuecomment-1515475367>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A4ABJWXSW6JGI4T5MI7DEZ3XCBTH3ANCNFSM6AAAAAAXD5ANNY>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

KeatonSushi commented:
I should be home around 4 or 5pm

On Wed, Apr 19, 2023, 5:45 PM Alek ***@***.***> wrote:

> I dont see it fixed yet. We can connect tomorrow if you like.
>
> > On Apr 19, 2023, at 6:35 PM, KeatonSushi ***@***.***> wrote:
> >
> >
> > I fixed the navbar.
> >
> > On Wed, Apr 19, 2023, 4:52 PM Keaton Milliner ***@***.***> wrote:
> >
> > > Or where should I copy it from rather.
> > >
> > > On Wed, Apr 19, 2023, 4:51 PM Keaton Milliner ***@***.***>
> > > wrote:
> > >
> > >> Oh okay. How do I copy and paste your original navbar?
> > >>
> > >> On Wed, Apr 19, 2023, 4:49 PM Alek ***@***.***> wrote:
> > >>
> > >>> Sorry I’m not understanding. I have a navbar set up that is shared
> > >>> across all screens, it looks like you’re implementing a new nav bar
> in a
> > >>> page called AboutUsNew.html
> > >>>
> > >>> > On Apr 19, 2023, at 5:43 PM, KeatonSushi ***@***.***> wrote:
> > >>> >
> > >>> >
> > >>> > I did a copy and paste from the index.html file you had.
> > >>> >
> > >>> > On Wed, Apr 19, 2023, 4:32 PM Alek ***@***.***> wrote:
> > >>> >
> > >>> > > ***@***.**** commented on this pull request.
> > >>> > > ------------------------------
> > >>> > >
> > >>> > > In aboutUsNew.html
> > >>> > > <
> > >>>
> https://github.com/LitterPic/litterpic_web/pull/7#discussion_r1171875248
> > >>> >
> > >>> > > :
> > >>> > >
> > >>> > > > - <ul class="navbar-nav ms-auto">
> > >>> > > - <li class="nav-item">
> > >>> > > - <a class="nav-link" aria-current="page" href="#">Home</a>
> > >>> > > - </li>
> > >>> > > - <li class="nav-item">
> > >>> > > - <a class="nav-link" href="#">Events</a>
> > >>> > > - </li>
> > >>> > > - <li class="nav-item">
> > >>> > > - <a class="nav-link" href="#">Donate</a>
> > >>> > > - </li>
> > >>> > > - <li class="nav-item">
> > >>> > > - <a class="nav-link" href="#">About Us</a>
> > >>> > > - </li>
> > >>> > > - </ul>
> > >>> > > - </div>
> > >>> > > + <div class="container-fluid">
> > >>> > >
> > >>> > > I have a navbar.html and an include-navbar.js defined. Are you
> > >>> defining a
> > >>> > > new navbar?
> > >>> > >
> > >>> > > —
> > >>> > > Reply to this email directly, view it on GitHub
> > >>> > > <
> > >>>
> https://github.com/LitterPic/litterpic_web/pull/7#pullrequestreview-1392914761
> > >>> >,
> > >>> > > or unsubscribe
> > >>> > > <
> > >>>
> https://github.com/notifications/unsubscribe-auth/A4ABJWWLZKPWFNIOZWE63UDXCBKWHANCNFSM6AAAAAAXD5ANNY
> > >>> >
> > >>> > > .
> > >>> > > You are receiving this because you authored the thread.Message
> ID:
> > >>> > > ***@***.***>
> > >>> > >
> > >>> > —
> > >>> > Reply to this email directly, view it on GitHub <
> > >>>
> https://github.com/LitterPic/litterpic_web/pull/7#issuecomment-1515423456
> >,
> > >>> or unsubscribe <
> > >>>
> https://github.com/notifications/unsubscribe-auth/AZYPMLYKJSQJWIMAO27FQN3XCBL7NANCNFSM6AAAAAAXD5ANNY
> > >>> >.
> > >>> > You are receiving this because you commented.
> > >>> >
> > >>>
> > >>> —
> > >>> Reply to this email directly, view it on GitHub
> > >>> <
> https://github.com/LitterPic/litterpic_web/pull/7#issuecomment-1515429448
> >,
> > >>> or unsubscribe
> > >>> <
> https://github.com/notifications/unsubscribe-auth/A4ABJWWSRXP2UTD7FCGSTDDXCBMXBANCNFSM6AAAAAAXD5ANNY
> >
> > >>> .
> > >>> You are receiving this because you authored the thread.Message ID:
> > >>> ***@***.***>
> > >>>
> > >>
> > —
> > Reply to this email directly, view it on GitHub <
> https://github.com/LitterPic/litterpic_web/pull/7#issuecomment-1515468072>,
> or unsubscribe <
> https://github.com/notifications/unsubscribe-auth/AZYPMLYCO3SYW4THTOJ3CM3XCBSE5ANCNFSM6AAAAAAXD5ANNY
> >.
> > You are receiving this because you commented.
> >
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/LitterPic/litterpic_web/pull/7#issuecomment-1515475367>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A4ABJWXSW6JGI4T5MI7DEZ3XCBTH3ANCNFSM6AAAAAAXD5ANNY>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
